--- serializers.py.orig	2017-05-08 16:09:38.931707352 +0000
+++ serializers.py	2017-05-08 16:12:22.540663461 +0000
@@ -96,9 +96,9 @@
     def _check_user(self, payload):
         username = jwt_get_username_from_payload(payload)
 
-        if not username:
-            msg = _('Invalid payload.')
-            raise serializers.ValidationError(msg)
+        #if not username:
+        #    msg = _('Invalid payload.')
+        #    raise serializers.ValidationError(msg)
 
         # Make sure user exists
         try:
